content/tutorial-rippled-setup.md

```diff
@@ -75,9 +77,9 @@ A `rippled` server should run comfortably on commodity hardware, to make it inex
 - Disk: Minimum 50GB SSD recommended (500+ IOPS, more is better) for the database partition
 - RAM: 4+GB
```

Comment:
Let's update the minimum requirements to 8GB, to match the comments in the capacity planning doc.

Reply:
The capacity planning does not describe the minimum settings, but the production deployment settings. I think we should describe both, and more, and those will be better organized in the documentation when we make hardware/configuration suggestions for the different persona/use cases. In the mean time, I thought this struck a healthy middle ground of having a minimum requirements section and capacity planning recommendations. What do you think, @mDuo13 ?

Reply:
I think that what I've read recently is that 4GB is not good enough for the minimum requirements anymore given growth in the ledger size and transaction volume